src/mod.ts

```diff
@@ -1,4 +1,4 @@
-import { parse } from "./deps.ts";
+import { parseArgs } from "@std/cli/parse-args";
```

Comment:
Nice! no more external dependencies?
Should we provide an std version?

Reply:
Ohh just saw this gets resolved in the [deno.jsonc](https://github.com/slackapi/deno-slack-protocols/pull/8/files#diff-81342196266f9373572c759839aa5a9ab9942af52ce56316b4855b3df8db588aR19)

Reply:
@WilliamBergamin yeah, the other option is to use an `import_map.json` file instead of putting this inside `deno.jsonc`. I have a feeling the Deno team is converging on putting everything into `deno.jsonc`, though.